scripts/config.example.json

```diff
@@ -1,5 +1,6 @@
 {
 	"ips": {
+		"adguardhome_upstream": ["tls://dns.google","https://dns.google/dns-query"],
```

Comment:
I'm not familar with adguardHome, does it support non DoH servers, if so, would be good to have an example of that here?

Reply:
Hello, that is already mentioned on the same page, it supports much more than just DoH servers: https://github.com/AdguardTeam/AdGuardHome/wiki/Configuration#upstreams

But yeah, I like your idea.